guides/source/command_line.md

```diff
@@ -21,13 +21,20 @@ There are a few commands that are absolutely critical to your everyday usage of
 
 * `rails console`
 * `rails server`
-* `bin/rails`
 * `rails generate`
 * `rails dbconsole`
 * `rails new app_name`
 
 All commands can run with `-h` or `--help` to list more information.
 
+There is also a set of commands that in previous versions of Rails used to be invoked with `rake`. Since Rails 5.0+ `rails` is the new default for running these commands. Again, based on their individual predicted frequency of use, some of them are:
```

Comment:
Sorry to be difficult, but I was thinking of something a bit more drastic: this changed 3 versions ago; I don't think this documentation should include any indication which commands were once rake tasks, or even mention that change at all. The below items should be mixed into the above list, and the subheadings under "Former rake commands" should be top-level headings in their own right.